gprofiler/profilers/python.py

```diff
@@ -82,6 +85,50 @@ def _add_versions_to_stacks(
     return result
 
 
+class PythonMetadta(ApplicationMetadata):
+    _PYTHON_VERSION_TIMEOUT = 3
+
+    @classmethod
+    def _get_python_version(cls, process: Process, stop_event: Event) -> str:
```

Comment:
Kind of duplicates the logic in `py_module_version.py`. If we execute Python anyway it's better to use this function there as well.

Reply:
Mhm

Reply:
You're right but this code will be consolidated in a later PR